pkg/process/v6/transformers/os/transform.go

```diff
@@ -232,12 +234,18 @@ func getPackage(group groupIndex) *grypeDB.Package {
 	}
 }
 
-func getOSInfo(group string) (string, string, string) {
+func getOSInfo(group string) (string, string, string, string) {
```

Comment:
non-blocking: this is getting to be enough return values of the same type that returning a struct might make sense and be more readable.